logger.go

```diff
@@ -44,6 +44,8 @@ Configuration
 
 SENTRY_DSN: If provided, warning and error logs will be sent to sentry.
 
+SENTRY_ENVIRONMENT: If provided, used to identify the error or warning log source.
```

Comment:
Shouldn't this this variable be mandatory?

Reply:
This is not required in the scope of this library